IntegrationTests/Runner/Scenarios/URLSession/URLSessionScenarios.swift

```diff
@@ -20,22 +20,18 @@ private class InheritedURLSessionDelegate: DDURLSessionDelegate {
 /// An example of instrumenting existing `URLSessionDelegate` with `DDURLSessionDelegate` through composition.
 private class CompositedURLSessionDelegate: NSObject, URLSessionTaskDelegate, URLSessionDataDelegate, __URLSessionDelegateProviding {
```

Comment:
this type is just like another delegate impl.

Reply:
Does it make sense to remove it now?

Reply:
we will keep this until the deletion of old mechnism because new solutions gives smoother migration path.

`__URLSessionDelegateProviding` will be marked deprecated though as a separate PR.